web/public/mgmt/config.yml

```diff
@@ -1500,14 +1500,22 @@ collections:
           - label: Weight
             name: weight
             widget: number
-          - label: Task
+          - label: Task (this or License Task is required)
```

Comment:
[Sand] I might be wrong, but I think in Netlify, all of the fields are required by default, I think you need to pass in an attribute where are you set required to false

Reply:
Based on what I'm seeing in the docs I think it does default to optional, but being explicit wouldn't hurt especially in this case.

Reply:
I think you're right @FarazA22 , I will make them not required